doc/_includes/header.html

```diff
@@ -41,18 +41,18 @@
 
       <section id="menu">
         <ul>
-          <li><a href="{{ site.baseurl }}index.html">Introduction</a></li>
-          <li><a href="{{ site.baseurl }}news/">News</a></li>
-          <li><a href="{{ site.baseurl }}install.html">Installation</a></li>
-          <li><a href="{{ site.baseurl }}language.html">Schema Language</a></li>
-          <li><a href="{{ site.baseurl }}encoding.html">Encoding</a></li>
-          <li><a href="{{ site.baseurl }}rpc.html">RPC Protocol</a></li>
-          <li><a href="{{ site.baseurl }}capnp-tool.html">The <code>capnp</code> Tool</a></li>
-          <li><a href="{{ site.baseurl }}cxx.html">C++ Serialization</a></li>
-          <li><a href="{{ site.baseurl }}cxxrpc.html">C++ RPC</a></li>
-          <li><a href="{{ site.baseurl }}otherlang.html">Other Languages</a></li>
-          <li><a href="{{ site.baseurl }}roadmap.html">Road Map</a></li>
-          <li><a href="{{ site.baseurl }}faq.html">FAQ</a></li>
+          <li><a href="{{ site.baseurl }}index.html"><p>Introduction</p></a></li>
```

Comment:
The reason to use a `p` tag here seems to be so that the generated DOM content (below in JS) creates the same DOM tree as the hard-coded HTML here.

I can't say I love the `p` tag.

I wonder if you can:

- Avoid it entirely, or

- Use a `span` to indicate that this has no particular meaning as an element